images/minimal-notebook/setup-scripts/setup_julia.py

```diff
@@ -36,7 +36,10 @@ def get_latest_julia_url() -> tuple[str, str]:
         "https://julialang-s3.julialang.org/bin/versions.json"
     ).json()
     stable_versions = {k: v for k, v in versions.items() if v["stable"]}
-    latest_version_files = stable_versions[max(stable_versions)]["files"]
```

Comment:
Thanks!
I think it is better to add comparator: `key=lambda ver: [int(sub_ver) for sub_ver in ver.split(".")]` as sort argument.

Funny fact: I actually thought about this the second time I was writing such a script, https://github.com/jupyter/docker-stacks/pull/2075/files